src/pseudo/PspUpf.jl

```diff
@@ -0,0 +1,200 @@
+using LinearAlgebra
+using Interpolations: linear_interpolation
+using PseudoPotentialIO: load_upf
+
+struct PspUpf{T,IT} <: NormConservingPsp
+    Zion::Int                             # Ionic charge (Z - valence electrons)
+    lmax::Int                             # Maximal angular momentum in the non-local part
+
+    rgrid::Vector{T}                      # Radial grid
+    drgrid::Vector{T}                     # Radial grid derivative / integration factor
```

Comment:
It is; naming is a huge mess though. In `PseudoPotentialsIO.jl`, I'm using the naming convention from the `upf2json` python code which is not the same as what's in the UPF file, probably because the names in the UPF file are not particularly helpful in all cases.

Reply:
Maybe have a longer docstring or a comment block of explanaition of the naming convention?